person.rb

```diff
@@ -19,6 +20,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rentals << rental
+    rental.person = self
+  end
+
```

Comment:
![image](https://github.com/fpsapc/OOPSchoolLibrary/assets/93039359/87e1b33a-d6ec-4c75-b9a7-c2c6046e6a63)


- [ ] It would be nice if you fix your add_rental method because it is giving us an error as you can see in the image above Kindly fix it to make your app work as expected 👍